examples/environment-variables/pages/index.js

```diff
@@ -90,4 +85,18 @@ const IndexPage = () => (
   </div>
 )
 
+// `getStaticProps`, and similar Next.js methods like `getStaticPaths` and `getServerSideProps`
+// only run in the server. Check the terminal to see the environment variables
+export async function getStaticProps() {
+  // Using the variables below in the browser will return `undefined`. Next.js doesn't
+  // expose environment variables unless they start with `NEXT_PUBLIC_`
+  console.log('[Server only] ENV_VARIABLE:', process.env.ENV_VARIABLE)
```

Comment:
We refer to the environment as `Node.js environment` instead of `Server` as otherwise it's unclear when it runs